app/models/megaphone/episode.rb

```diff
@@ -331,8 +332,8 @@ def get_media_info(enclosure)
         info[:location] = resp.env.response_headers["location"]
         info[:size] = resp.env.response_headers["content-length"].to_i
       else
-        logger.error("DTR media redirect not returned: #{resp.status}", enclosure: enclosure, resp: resp)
-        raise("DTR media redirect not returned: #{resp.status}")
+        logger.error("DTR media redirect not returned: #{resp&.status}", enclosure: enclosure, headers: resp&.env&.response_headers)
```

Comment:
the Faraday response includes a reference back to the Faraday response, i.e. `response == response.env.response`, which was causing an infinite loop / stack error when serializing to json.

Reply:
Ooh, fun!